src/pen.lisp

```diff
@@ -15,11 +15,11 @@
   (curve-steps 100))
 
 (defmacro with-pen (pen &body body)
-  (alexandria:once-only (pen)
```

Comment:
Good catch, turns out this is from when the code looked like this: https://github.com/vydd/sketch/commit/c8af25403504f7314ba17ea95a7744c430ea176f#diff-c0fb98a0dfb7b4685052b4310125588ec7a38261f94e452eaa0675d24a5212faR21-R22